src/lib/cache/subCache.cpp

```diff
@@ -735,34 +786,35 @@ void subCacheItemInsert(CachedSubscription* cSubP)
 */
 void subCacheItemInsert
 (
-  const char*                        tenant,
-  const char*                        servicePath,
-  const ngsiv2::HttpInfo&            httpInfo,
-  const ngsiv2::MqttInfo&            mqttInfo,
-  const std::vector<ngsiv2::EntID>&  entIdVector,
-  const std::vector<std::string>&    attributes,
-  const std::vector<std::string>&    metadata,
-  const std::vector<std::string>&    conditionAttrs,
-  const char*                        subscriptionId,
-  int64_t                            expirationTime,
-  int64_t                            maxFailsLimit,
-  int64_t                            throttling,
-  RenderFormat                       renderFormat,
-  int64_t                            lastNotificationTime,
-  int64_t                            lastNotificationSuccessTime,
-  int64_t                            lastNotificationFailureTime,
-  int64_t                            lastSuccessCode,
-  const std::string&                 lastFailureReason,
-  StringFilter*                      stringFilterP,
-  StringFilter*                      mdStringFilterP,
-  const std::string&                 status,
-  double                             statusLastChange,
-  const std::string&                 q,
-  const std::string&                 geometry,
-  const std::string&                 coords,
-  const std::string&                 georel,
-  bool                               blacklist,
-  bool                               onlyChanged
+  const char*                             tenant,
+  const char*                             servicePath,
+  const ngsiv2::HttpInfo&                 httpInfo,
+  const ngsiv2::MqttInfo&                 mqttInfo,
+  const std::vector<ngsiv2::EntID>&       entIdVector,
+  const std::vector<std::string>&         attributes,
+  const std::vector<std::string>&         metadata,
+  const std::vector<std::string>&         conditionAttrs,
+  const std::vector<ngsiv2::SubAltType>&  altTypes,
```

Comment:
The only actual modification in this block (other modified lines are just indent adjustment)

NTC (informative)